theme/templates/includes/res_type_col_community.html

```diff
@@ -0,0 +1,100 @@
+{% load hydroshare_tags %}
```

Comment:
This file seems like a duplicate of https://github.com/hydroshare/hydroshare/blob/develop/theme/templates/includes/res_type_col.html

Consider reusing the original

Reply:
The resource objects use newly optimized queries and so the display breaks unless split out into separate code. I'd like to see the older code optimized in a separate PR, tested then the files can be consolidated. Overall, this file could possibly find a home as a part of another pre-existing file if it's not all too long. It's hard to know in this area how many ways to split things out, versus having colossal files.

Reply:
Not sure I follow. Isn't this file identical to the one I mention above? If so it can be reused using Django template language. You can then point your script to your instance using specific selectors.

Reply:
It's not identical.

Reply:
Is it possible to apply the same optimizations to the older code and then reuse the files?

Reply:
My understanding is no. Also, this is the more maintainable approach that allows for independent updates (and later consolidation). Groups code needs to be refactored as a part of a different PR.